### DISCUSSION TITLE
Strapi on Netlify

### DISCUSSION THREAD

**Discussion by piyushpaliwal** [strapi/strapi#6460] [Legacy Topics]
I am familiar with Netlify and understand that it is primarily used for hosting static sites, however there are multiple places where I have found mention of Netlify with regards to Strapi.

The most recent one is the deployment options session that you guys published on [youtube](https://www.youtube.com/watch?v=Rw7FTdhbgI8&t=3575s) - see screenshot below
![image](https://user-images.githubusercontent.com/635693/83334504-9c6e8580-a2c4-11ea-9a01-f7c9f7c996dd.png)

The official talk clearly shows Netlify under PaaS options. Can someone point me in the direction of how this can be done?

**Comment by robertpiosik** [comment]
Hey, Netlify is a PaaS. It has something called [Netlify Functions](https://www.netlify.com/products/functions/) where you can run code in multiple languages among whom is JavaScript, upon which Strapi was written (it's simply a Node.js application built with Koa).

Netlify doesn't reinvent the wheel though. They're using available cloud providers under the hood so they are basically selling what they offer with conveniences of simpler management and an additional fee.

**Reply by piyushpaliwal** [reply]
That makes sense. All this while, I was just utilizing Netlify for static websites. Is there any guide/instruction set available which I can refer and get a working strapi environment with Netlify?

**Reply by robertpiosik** [reply]
I would just google about running a Node.js code on this platform. It will guide you well.
